lib/styles/settings/_colors.scss

```diff
@@ -31,7 +31,7 @@ $color-ice-orange: #ffe6ca;
 $color-gigas-violet: #62378d;
 $color-deluge-violet: #7b569f;
 
-$color-total-black: #000;
+//$color-total-black: #000; //DEPRECATED
```

Comment:
why is it commented out? I believe all these colors defined in this file are deprecated

Reply:
Yes, I will revert and not touch this file more.